planning/obstacle_cruise_planner/include/obstacle_cruise_planner/common_structs.hpp

```diff
@@ -110,14 +110,16 @@ struct StopObstacle : public TargetObstacleInterface
     const std::string & arg_uuid, const rclcpp::Time & arg_stamp,
     const geometry_msgs::msg::Pose & arg_pose, const Shape & arg_shape,
     const double arg_lon_velocity, const double arg_lat_velocity,
-    const geometry_msgs::msg::Point arg_collision_point)
+    const geometry_msgs::msg::Point arg_collision_point, const double arg_dist_to_collide)
   : TargetObstacleInterface(arg_uuid, arg_stamp, arg_pose, arg_lon_velocity, arg_lat_velocity),
     shape(arg_shape),
-    collision_point(arg_collision_point)
+    collision_point(arg_collision_point),
```

Comment:
Is it hard to remove the collision_point here?
If this still exists here, it will be used for some calculation by mistake.

Actually, the nearest search is applied to this collision point, which is not expected.
https://github.com/autowarefoundation/autoware.universe/blob/cb409fa58f516f8499b6970820c39f4de345ac11/planning/obstacle_cruise_planner/src/planner_interface.cpp#L407

Reply:
If this variable has to exist for the debug information, we'd better change the variable name to `collision_point_for_debug` for example so that the developer can understand the reason for this variable to exist.